src/app/shared/stache/modules/config/config.service.ts

```diff
@@ -0,0 +1,18 @@
+import { Inject, OpaqueToken } from '@angular/core';
+
+import { StacheConfigTransformer } from './config-transformer';
+
+export let STACHE_CONFIG_TRANSFORMERS = new OpaqueToken('Injection token for config transformers.');
```

Comment:
We'll need to update the Opaque Token to Inject Token when we upgrade to angular 4. https://blog.thoughtram.io/angular/2016/05/23/opaque-tokens-in-angular-2.html#injectiontoken-since-angular-4x